library/choose_disk.py

```diff
@@ -606,7 +626,7 @@ def main():
     matched_devices = None
     lookup_disks = None
 
-    setup_logging()
+    setup_logging("/tmp/choose_disk.log")
```

Comment:
Please don't use a hard coded path for logging. This should be either a well known/established location or at the very least a configurable one. In systems where `/tmp/` doesn't exist, this would break as the `setup_logging` function doesn't deal with any errors for log locations

Reply:
This should not have been commited .... Wrong staging here. Fixed by 508145e9128fe375c7a3cd8a57f1a3af4e6d7d2d

Thx !